README.md

```diff
@@ -200,9 +211,77 @@ Security Team will need to update the project release node on backdropcms.org to
 inidciate that a release is a security release.
 
 
+The Backdrop Bug Squad
+----------------------
+
+The `Backdrop Bug Squad` consistes of trusted members of the backdrop-contrib
+group who will help other contributors stay on top of minor bug fixes and User
+Interface improvements.
+
+Responsibilities include:
+- Reviewing Pull Requests for eligibility.
+- Merging eligible Pull Requests.
+- Communicating with existing maintainers.
+
+Fixes eligible inlude:
+- Minor Bugs (Fatal errors, PHP notices, validation errors, etc).
+- Minor User Interface changes (typos, spelling, grammar, terminology, etc).
+- Any packaging issue preventing the project from being downloaded and installed
+  via the Project Browser.
+- Any change that could be accepted after a feature freeze.
+
+Fixes NOT eligible include:
+- Any new feature.
+- Any major code change or addition.
+
+Any eligable Pull Request must be tagged as "Reviewed and Tested By the
+Community" for a period of at least 2 weeks before it can be merged.
+
+You may apply to become a member of the Backdrop Bug Squad. Please
```

Comment:
...we need a list of current members of the Squad in b.org to be linked from here (either on https://backdropcms.org/leadership or a sub-page of that). Perhaps ideally an email set up or other means of people to be able to communicate with the Squad.

Reply:
Can we link to https://github.com/orgs/backdrop-ops/teams/bug-squad/members? Since this is a GitHub thing, I think we should link to GitHub instead of b.org.

Reply:
People need to have a GitHub account to view that page (just tried incognito and it asked fr my credentials), so I think that it would be good to have on b.org too.